docs/content/preview/secure/tls-encryption/server-certificates.md

```diff
@@ -229,87 +235,74 @@ Now you can generate the node key `node.key` and node certificate `node.crt` for
 
 ### Generate configuration for each node
 
-Repeat the steps in this section once for each node.
-The IP address of each node is `<node-ip-address>`
-
-1. Generate a configuration file (`node.conf`) for a node, using the node's IP address (`<node-ip-address>`) as the directory name.
-
-```sh
-$ cat > <node-ip-address>/node.conf
-```
-
-2. Add the following sample configuration content (use as-is, or customize as needed).
+Repeat the following steps for each node, replacing `<node-ip-address>` with the IP address of the subject node:
 
-{{< note title="Note" >}}
+1. Generate a configuration file `node.conf` for a node, using the node's IP address `<node-ip-address>` as the directory name, as follows:
 
-Remember to replace the `<node-ip-address>` entry in the example configuration file below with the IP address for each node.
+   ```sh
+   cat > <node-ip-address>/node.conf
+   ```
 
-{{< /note >}}
+2. Add the following sample configuration content (use as-is, or customize as needed):
 
-```sh
-#################################
-# Example node configuration file
-#################################
-
-[ req ]
-prompt=no
-distinguished_name = my_distinguished_name
-
-[ my_distinguished_name ]
-organizationName = Yugabyte
-# Required value for commonName, do not change.
-commonName = <node-ip-address>
-```
+   ```sh
+   #################################
+   # Example node configuration file
+   #################################
+   
+   [ req ]
+   prompt=no
+   distinguished_name = my_distinguished_name
+   
+   [ my_distinguished_name ]
+   organizationName = Yugabyte
+   # Required value for commonName, do not change
+   commonName = <node-ip-address>
+   ```
```

Comment:
```suggestion
   ```sh
   #################################
   # Example node configuration file
   #################################
   
   [ req ]
   prompt=no
   distinguished_name = my_distinguished_name
   
   [ my_distinguished_name ]
   organizationName = Yugabyte
   # Required value for commonName, do not change
   commonName = <node-ip-address>
   
  # Multiple subject alternative names (SANs) such as IP Address, DNS Name, Email, URI, etc can be specified under the following section
  
   [ req_ext]
   SubjectAltName = @alt_names
   [alt_names]
   IP.1 = <IP Address>
   IP.2 = <IP Address>
   DNS.1 = < DNS Name> 
   DNS.2 = < DNS Name>
   ```
```

Reply:
@lizayugabyte I have included an extension that allows specifying subject alternative names for using multiple names within a single certificate in the above node configuration file

Reply:
Changed